package_plugin.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-pyuic5 ui_files/config.ui -o crowd_anki/config/config_ui.py
+bash generate_ui.sh
```

Comment:
if you make it executable and add shebang, you should be able to call it similar to the fetch dependencies one

Reply:
Oh you can do that anyways, I just always use bash for some reason. Changed it to be simpler, done :+1: